### PR TITLE
CLDC-2741 Order log search results

### PR DESCRIPTION
**Pull request by kosiakkatrina**

### CODE REVIEW

File: app/models/lettings_log.rb [hunk 1]
```diff
@@ -58,11 +58,17 @@ class LettingsLog < Log
     query.all
   }
   scope :search_by, lambda { |param|
+    by_id = Arel.sql("case when lettings_logs.id = #{param.to_i} then 0 else 1 end")
```
Comment (natdeanlewissoftwire) [inline app/models/lettings_log.rb]: do we want an ILIKE for id too?

Comment (natdeanlewissoftwire) [reply app/models/lettings_log.rb]: also, as these are SQL I'd personally rather they all be upper case (or at least a consistent case) - though I'm not sure this is the case thus far in the codebase so I don't have a problem if you prefer it this way :)

**Review by natdeanlewissoftwire — APPROVED** [review]
lgtm!

File: app/models/lettings_log.rb (hunk 1 above)
Comment (kosiakkatrina) [reply app/models/lettings_log.rb]: I think we only search IDs for exact matches, so don't think it needs to order by similarity. Good point on casing!

**Pull request merged**